webapp/channels/src/components/channel_invite_modal/channel_invite_modal.tsx

```diff
@@ -86,20 +103,79 @@ export default class ChannelInviteModal extends React.PureComponent<Props, State
         super(props);
         this.state = {
             values: [],
```

Comment:
It takes some mental energy to understand what `values` are.

Reply:
Renamed this to `selectedUsers` and changed `optionValues` to `groupAndUserOptions`